packages/sheet/types/index.d.ts

```diff
@@ -15,7 +15,7 @@ export class StyleSheet {
   isSpeedy: boolean
   ctr: number
   tags: Array<HTMLStyleElement>
-  container: HTMLElement
+  container: Node
   key: string
   nonce?: string
   before?: Element | null
```

Comment:
This probably should be adjusted as well, thoughts?
```suggestion
  before?: ChildNode | null
```

Reply:
Agree with this change. Without this change we will probably get TS errors in the `ts-migration` branch because `Node['firstChild']` is `ChildNode`, not `Element`.